### PR TITLE
Fixed hard coded package name on MarkdownPreview.py,

### PR DESCRIPTION
**Pull request by evandrocoan**
Hard coding the plugin name breaks the package if it renamed.

### CODE REVIEW

File: MarkdownPreview.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def load_resource(name):
 
     try:
         if is_ST3():
-            return sublime.load_resource('Packages/Markdown Preview/{0}'.format(name))
+            return sublime.load_resource('Packages/{}/{}'.format(__package__, name))
```
Comment (evandrocoan) [inline MarkdownPreview.py]: Note, I can only use `__package__` on Sublime Text 3. If this is meant to be compatible with Sublime Text 2, I need use this:
```
CURRENT_PACKAGE_FILE   = os.path.dirname( os.path.realpath( __file__ ) )
PACKAGE_ROOT_DIRECTORY = CURRENT_PACKAGE_FILE.replace( ".sublime-package", "" )
CURRENT_PACKAGE_NAME   = os.path.basename( PACKAGE_ROOT_DIRECTORY )
return sublime.load_resource('Packages/{}/{}'.format(CURRENT_PACKAGE_NAME   , name))
```

**Comment by facelessuser** [comment]
Why are you renaming the package?

**Comment by evandrocoan** [comment]
Because I liked a new name. On this, I am setting a standard into all packages I have installed to not have any spaces on their names, and as this package is named `Markdown Preview`, I renamed it to `MarkdownPreview`. 

1. By this standard I also renamed my GitHub fork to `MarkdownPreview`, then to install install this package I can just do `git clone URL` instead of `git clone URL "Markdown Preview"`.
1. With a name without spaces I can import something on this packages just by `from MarkdownPreview.file_name import cool_stuff`.
1. Other than that, you rename something when they name seems or is not fulfilling the purpose of:
    i. Describing correctly what does it represents.
    ii. It does represents, but you can never remember it by its name

Why do you rename?

**Comment by facelessuser** [comment]
> Why do you rename?

I was just asking for a legitimate reason.  I'll be honest, what happens on other people's fork is of little concern to me.  Also when people are doing non-standard things like renaming a package that is expected to be installed a certain way, it is also of little concern to me. But, this seems okay for ST3 and costs us nothing to implement, so I'm okay with this once we are no longer releasing for ST2.

The next release I believe is planned to be ST3 only, so I can probably add this on that branch.

**Comment by facelessuser** [comment]
New supported repo is here: https://github.com/facelessuser/MarkdownPreview. If this is still desired to be added, please issue a pull there and make sure this works in zipped sublime-package bundles as well.

FYI, the new packages will be named Markdown preview (with no spaces) moving forward.

**Pull request closed without merging**